src/index.css

```diff
@@ -11,3 +27,288 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/* header styles */
+header {
+  background-color: #d9ed92;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  
+}
+
+header .wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+
+h1, h2 {
+  font-family: 'Ovo', serif;
+  color: #168aad;
+}
+
+h1 {
+  font-size: 5rem;
+}
+
+h2 {
+  font-size: 2.5rem;
+  margin: 0;
+  padding: 20px 0;
+  text-align: center;
+}
+
+h3 {
+  font-size:  1.8rem;
+  font-family: 'Nanum Gothic', sans-serif;
+  padding: 0px 0 10px 0;
+}
+
+p {
+  font-family: 'Nanum Gothic', sans-serif;
+}
+
+.title {
+  position: relative;
+  z-index: 0;
+}
+
+.leaf {
+  position: absolute;
+  top: -125px;
+  left: 200px;
+  color: #76c893;
+  font-size: 350px;
+  transform: rotate(50deg);
+  z-index: -10;
+}
+
+/* header styles end */
+
+/* form styles */
+.formEntry {
+  display: flex;
+  text-align: center;
+  flex-direction: column;
+  background-color: #76c893;
+}
+
+header h1, header h2 {
+  text-align: left;
+}
+
+.instructions {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 auto;
+  padding: 15px;
+  width: 50%;
+  border-radius: 8px;
+  background-color: rgba(48, 45, 45, 0.37);
+}
+
+.instructions p {
```

Comment:
The instructions and form overflows at small screen sizes (320px)

Reply:
The inputs are also very wide at small screen sizes (looks like they are full width of the wrapper)